### DISCUSSION TITLE
회원정보

### DISCUSSION THREAD

**Discussion by zeus6768** [woowacourse-teams/2024-code-zap#434] [결정 사항]
1. ID
2. ID, username(필수, 중복 불가)
3. ID, username(필수, 중복 가능)
4. ID, username(선택, 중복 불가)
5. ID, username(선택, 중복 가능)
6. OAuth 2.0

**Comment by vi-wolhwa** [comment]
OAuth 이든, 이메일 인증이든 결국은 나중에 해야 한다고 생각합니다.
=> 무분별한 회원가입 제한

나중을 생각하면 username을 (필수/중복불가)로 하는 것이 맞다고 생각합니다.
아니면 id 없이 usernmae만
